LibrarySystemBackend/LibrarySystem/LibrarySystem/Services/StaffService.cs

```diff
@@ -29,7 +29,9 @@ public async Task<bool> AddStaffAsync(StaffViewModel staffViewModel)
                     StaffName = staffViewModel.StaffName,
                     StaffEmail = staffViewModel.StaffEmail,
                     EnrollmentDate = staffViewModel.EnrollmentDate,
-                    StaffID = staffViewModel.StaffID
+                    Id = staffViewModel.StaffID,
```

Comment:
Do not need to add Id to Add staff method (because it is an auto-generated value)